homeassistant/components/tibber/sensor.py

```diff
@@ -261,6 +300,29 @@ def __init__(self, tibber_home, sensor_name, device_class, unit, initial_state):
         self._attr_state = initial_state
         self._attr_unique_id = f"{self._tibber_home.home_id}_rt_{self._sensor_name}"
         self._attr_unit_of_measurement = unit
+        self._attr_state_class = state_class
+        if sensor_name in [
+            "last meter consumption",
+            "last meter production",
+        ]:
+            self._attr_last_reset = datetime.fromtimestamp(0)
+        elif self._sensor_name in [
+            "accumulated consumption",
+            "accumulated production",
+            "accumulated cost",
+        ]:
+            self._attr_last_reset = dt_util.as_utc(
+                dt_util.now().replace(hour=0, minute=0, second=0, microsecond=0)
```

Comment:
Should this be `dt_util.utcnow()` or is Tibber resetting based on the local timezone?

Reply:
Tibber resetting is based on the local timezone